### PR TITLE
[autoscaler] fix: available nodes grafana gauge color

### PR DESCRIPTION
**Pull request by samvantran**
**What type of PR is this?**
Fix

**What this PR does/ why we need it**:
Noticed this in the MWT. Previously it would show 100% available nodes in red and 80% as green.
This reverses the coloring. See screenshots:

Before:

Screen Shot 2020-06-16 at 1 37 49 PM

After: 

Screen Shot 2020-06-16 at 1 06 00 PM

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```
Not really worth calling out a color change on a grafana dashboard

**Checklist**

* [] The commit message explains the changes and why are needed.
* [] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ data:
       "editable": true,
       "gnetId": 3831,
       "graphTooltip": 0,
-      "id": null,
+      "id": 36,
```
Comment (gracedo) [inline stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml:61]: this needs to be `null`

Comment (samvantran) [inline stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml:61]:
```suggestion
      "id": null,
```

Comment (samvantran) [reply stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml]: i was wondering if that was going to have any effect. thanks, fixed in b19a677

Comment (gracedo) [reply stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml]: I always forget you can suggest commits, I shoulda done that lol

Comment (gracedo) [reply stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml]: np, we ran into this a lot in dc/os when people started contributing dashboards - the trick is remembering to check the "export for sharing externally" box
image

Comment (samvantran) [reply stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml]: ty 🙇

**Pull request merged**